x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings/latest_findings_table.tsx

```diff
@@ -49,7 +50,7 @@ const FindingsTableComponent = ({
 
   const pagination = useMemo(
     () =>
-      getEuiPaginationFromEsSearchSource({
+      getEuiPaginationFromEs({
         from,
         size,
         total: data?.total,
```

Comment:
eventually all tables should take the same props, `pagination` included. 

there's some inconsistency between pagination for `group-by-none` + `resource-findings` to pagination for `group-by-resource`, mainly that their `es` query and `ui` are different. once we sort this out we can iterate on this.